core/test/parser/test_parser.py

```diff
@@ -410,6 +412,10 @@ def test_pydantic_schema_data_update(
             load_schema("object_with_list_of_objects"),
             {"list": [{"members": {"a": None, "b": [], "c": {"d": "", "e": ""}}}]},
         ),
+        (
+            load_schema("object_with_list_of_objects"),
+            {"list": [{"a": None, "b": [], "c": {"d": "", "e": ""}}]},
+        ),
```

Comment:
are these different?

Reply:
yea, the second one matches the schema correctly.
First one is just incorrect schema obj

Reply:
let's remove the first one